### PR TITLE
make favicon customizable

### PR DESCRIPTION
**Pull request by willydouhard**

### CODE REVIEW

File: backend/chainlit/server.py [hunk 1]
```diff
@@ -112,8 +120,7 @@ async def watch_files_for_changes():
         os._exit(0)
 
 
-root_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-build_dir = os.path.join(root_dir, "frontend/dist")
+build_dir = os.path.join(PACKAGE_ROOT, "frontend/dist")
```
Comment (tpatel) [inline backend/chainlit/server.py:123]: Same feedback as my other comment
```suggestion
build_dir = os.path.join(PACKAGE_ROOT, "frontend", "dist")
```

File: backend/chainlit/db/__init__.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import os
 
-from chainlit.config import PACKAGE_ROOT, config
+from chainlit.config import BACKEND_ROOT, config
 from chainlit.logger import logger
 
-SCHEMA_PATH = os.path.join(PACKAGE_ROOT, "db/prisma/schema.prisma")
+SCHEMA_PATH = os.path.join(BACKEND_ROOT, "db/prisma/schema.prisma")
```
Comment (tpatel) [inline backend/chainlit/db/__init__.py:6]: Not sure if the slash could be a problem when running on Windows, here is a suggested change:
```suggestion
SCHEMA_PATH = os.path.join(BACKEND_ROOT, "db", "prisma", "schema.prisma")
```

File: backend/chainlit/server.py (hunk 1 above)
Comment (willydouhard) [reply backend/chainlit/server.py:123]: Good catch!

**Review by clementsirieix — APPROVED** [review]
🚀 LGTM

(Didn't know about the `join` caveat, good to know 👌)

**Pull request merged**